--- a/Makefile
+++ b/Makefile
@@ -411,16 +411,12 @@
 
 VER_STRING = $(VER_MAJOR).$(VER_MINOR).$(VER_RELEASE).$(BUILD_NUMBER) $(PACKAGE_SUFFIX)
 
-# load depenency packages
-include prebuilds.mk
-
 # name virtual targets
 .PHONY: all docs docsdox clean clean-build deep-clean build-before
 .PHONY: standard std-before std-after
 .PHONY: heavylog hvlog-before hvlog-after
 .PHONY: package clean-package deep-clean-package
 .PHONY: tools clean-tools deep-clean-tools
-.PHONY: clean-libexterns deep-clean-libexterns
 .PHONY: tests
 
 # dependencies tracking
@@ -465,7 +461,7 @@
 $(shell $(MKDIR) $(FOLDERS))
 
 # We need this file because we need git update
-build-before: libexterns deps/zlib/configure.log
+build-before: deps/zlib/configure.log
 
 std-before: build-before
 hvlog-before: build-before
@@ -476,12 +472,8 @@
 	VERSION=$(VER_STRING) $(DOXYTOOL) docs/doxygen.conf
 
 deep-clean: deep-clean-tools deep-clean-package
-	$(MAKE) -f libexterns.mk deep-clean-libexterns
 
-clean: submodule clean-build clean-tools clean-libexterns clean-package
-
-submodule:
-	-git submodule init && git submodule update
+clean: clean-build clean-tools clean-package
 
 clean-build:
 	-$(RM) $(STDOBJS) $(STD_MAIN_OBJ) $(filter %.d,$(STDOBJS:%.o=%.d)) $(filter %.d,$(STD_MAIN_OBJ:%.o=%.d))
@@ -564,14 +556,14 @@
 obj/cu/%.o: $(CU_DIR)/Sources/Basic/%.c
 	$(CPP) $(CXXFLAGS) $(CU_INC) -o"$@" "$<"
 
-obj/std/%.o obj/hvlog/%.o: src/%.cpp libexterns $(GENSRC)
+obj/std/%.o obj/hvlog/%.o: src/%.cpp $(GENSRC)
 	-$(ECHO) 'Building file: $<'
 	@grep -E "#include \"(\.\./)?(\.\./)?pre_inc.h\"" "$<" >/dev/null || echo "\n\nAll files should have #include \"pre_inc.h\" as first include\n\n" >&2 | false
 	@grep -E "#include \"(\.\./)?(\.\./)?post_inc.h\"" "$<" >/dev/null || echo "\n\nAll files should have #include \"post_inc.h\" as last include\n\n" >&2 | false
 	$(CPP) $(CXXFLAGS) -o"$@" "$<"
 	-$(ECHO) ' '
 
-obj/std/%.o obj/hvlog/%.o: src/%.c libexterns $(GENSRC)
+obj/std/%.o obj/hvlog/%.o: src/%.c $(GENSRC)
 	-$(ECHO) 'Building file: $<'
 	@grep -E "#include \"(\.\./)?(\.\./)?pre_inc.h\"" "$<" >/dev/null || echo "\n\nAll files should have #include \"pre_inc.h\" as first include\n\n" >&2 | false
 	@grep -E "#include \"(\.\./)?(\.\./)?post_inc.h\"" "$<" >/dev/null || echo "\n\nAll files should have #include \"post_inc.h\" as last include\n\n" >&2 | false
@@ -602,23 +594,12 @@
 
 tests: std-before $(TEST_BIN)
 
-libexterns: libexterns.mk
-	$(MAKE) -f libexterns.mk
-
-clean-libexterns: libexterns.mk
-	-$(MAKE) -f libexterns.mk clean-libexterns
-	-$(MAKE) -f enet.mk clean
-	-cd deps/zlib && $(MAKE) -f win32/Makefile.gcc clean
-	-cd deps/zlib && git checkout Makefile zconf.h
-	-$(RM) libexterns
-
 deps/centijson/src/json.c deps/centijson/src/value.c deps/centijson/src/json-dom.c: build-before
 deps/libspng/spng/spng.c: build-before
 deps/zlib/contrib/minizip/unzip.c deps/zlib/contrib/minizip/ioapi.c: build-before
 
 
 deps/zlib/configure.log:
-	git submodule sync && git submodule update --init
 	touch deps/zlib/configure.log
 	cd deps/zlib && ./configure --static
 
